### ISSUE TITLE
Eevee displacement setup : "BLENDER_EEVEE" not found in ('ALL', 'EEVEE', 'CYCLES')

### ISSUE THREAD

**Issue by optimus007** [Poly-Haven/polyhavenassets#48]
**Describe the bug**
On addon v1.0.7 & blender 3.4.1
after clicking setup displacement+confirming i get this error

**Console log**

```
 File "C:\Users\P\AppData\Roaming\Blender Foundation\Blender\3.4\scripts\addons\polyhavenassets\operators\tex_displacement_setup.py", line 81, in setup_mesh_displacement
    active_output = material.node_tree.get_output_node(context.scene.render.engine)
TypeError: ShaderNodeTree.get_output_node(): error with argument 1, "target" -  enum "BLENDER_EEVEE" not found in ('ALL', 'EEVEE', 'CYCLES')

```

**Comment by gregzaal** [comment]
Thanks for the report, will fix this first thing tomorrow 👍 Must have slipped through with some code optimizations.

**Comment by gregzaal** [comment]
Fixed in 8eaa6aa32c889d17fb88edaec6767a2f943a78f1 (v1.0.8)
